index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head><link rel="stylesheet" href="styles.css"></head>
+  <body>
+    
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <form id="survey-form">
+      <label for="name" id="name-label">Name</label><br>
+      <input id="name" type="text" name="name" placeholder="Enter your name" required></input><br><br>
+      <label for="email" id="email-label">Email</label><br>
+      <input id="email" type="email" name="email" pattern="[^ @]*@[^ @]*" placeholder="Enter your email" required></input><br><br>
```

Comment:
You have used &lt;br&gt; tags in many places to break the lines in your <body> which is unprofessional
you should use your CSS stylesheet and in that, you can use display: flex for this and flex-wrap commands using CSS.

Reply:
I know sometimes using a &lt;br&gt; tag in html is a easy way to break the lines but It would be nice and  professional if you use your CSS stylesheet to break the lines using commands like flex and flex-wrap. 😉